tsconfig.json

```diff
@@ -1,17 +1,37 @@
 {
   "compilerOptions": {
-    "module": "ESNext",
-    "target": "es6",
```

Comment:
Since we're targeting ESM, I think we want module to stay. What does the target change do?

Reply:
Target being ES6 is specifying ECMAScript version to emit (in this case ECMAScript 6). It won't affect esm/cjs.